<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="5942" started-at="2015-05-12T07:43:48Z" finished-at="2015-05-12T07:43:54Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="5942" started-at="2015-05-12T07:43:48Z" finished-at="2015-05-12T07:43:54Z">
      <class name="com.emar.test.testcases.yiqifabackend.mediapurchasingmanagement.SQE">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:com.emar.test.testcases.yiqifabackend.mediapurchasingmanagement.SQE@1c4be76]" name="beforeTest" is-config="true" duration-ms="13" started-at="2015-05-12T15:43:48Z" finished-at="2015-05-12T15:43:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="test(java.lang.String)[pri:0, instance:com.emar.test.testcases.yiqifabackend.mediapurchasingmanagement.SQE@1c4be76]" name="test" duration-ms="23" started-at="2015-05-12T15:43:49Z" data-provider="feeder" finished-at="2015-05-12T15:43:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[¹©Ó¦ÉÌ123]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method status="FAIL" signature="test_1_addSQE_success(java.lang.String, java.lang.String)[pri:0, instance:com.emar.test.testcases.yiqifabackend.mediapurchasingmanagement.SQE@1c4be76]" name="test_1_addSQE_success" duration-ms="3661" started-at="2015-05-12T15:43:49Z" data-provider="addSQE_success" finished-at="2015-05-12T15:43:53Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[SeleniumTest]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[2060]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[no such element
  (Session info: chrome=41.0.2272.76)
  (Driver info: chromedriver=2.11.298604 (75ea2fdb5c87f133a8e1b8da16f6091fb7d5321e),platform=Windows NT 6.1 SP1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 2.02 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.41.0', revision: '3192d8a', time: '2014-03-27 17:18:15'
System info: host: 'renshaohai-PC', ip: '172.16.41.74', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.7.0_60'
Session ID: 4f099125b0d2ce2f73d91f41e30148df
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{platform=XP, acceptSslCerts=true, javascriptEnabled=true, browserName=chrome, chrome={userDataDir=C:\Users\RENSHA~1\AppData\Local\Temp\scoped_dir10680_29761}, rotatable=false, locationContextEnabled=true, mobileEmulationEnabled=false, version=41.0.2272.76, takesHeapSnapshot=true, cssSelectorsEnabled=true, databaseEnabled=false, handlesAlerts=true, browserConnectionEnabled=false, nativeEvents=true, webStorageEnabled=true, applicationCacheEnabled=false, takesScreenshot=true}]]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: no such element
  (Session info: chrome=41.0.2272.76)
  (Driver info: chromedriver=2.11.298604 (75ea2fdb5c87f133a8e1b8da16f6091fb7d5321e),platform=Windows NT 6.1 SP1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 2.02 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.41.0', revision: '3192d8a', time: '2014-03-27 17:18:15'
System info: host: 'renshaohai-PC', ip: '172.16.41.74', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.7.0_60'
Session ID: 4f099125b0d2ce2f73d91f41e30148df
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{platform=XP, acceptSslCerts=true, javascriptEnabled=true, browserName=chrome, chrome={userDataDir=C:\Users\RENSHA~1\AppData\Local\Temp\scoped_dir10680_29761}, rotatable=false, locationContextEnabled=true, mobileEmulationEnabled=false, version=41.0.2272.76, takesHeapSnapshot=true, cssSelectorsEnabled=true, databaseEnabled=false, handlesAlerts=true, browserConnectionEnabled=false, nativeEvents=true, webStorageEnabled=true, applicationCacheEnabled=false, takesScreenshot=true}]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:193)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:145)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:595)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:348)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:445)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:357)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:340)
	at com.emar.test.initviews.yiqifabackend.mediapurchasing.MediaPurchasingManagementSystem_SQE.addSQEBtn(MediaPurchasingManagementSystem_SQE.java:33)
	at com.emar.test.testcases.yiqifabackend.mediapurchasingmanagement.SQE.test_1_addSQE_success(SQE.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:330)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_1_addSQE_success -->
        <test-method status="PASS" signature="test_2_addSQE_SQL(java.lang.String)[pri:0, instance:com.emar.test.testcases.yiqifabackend.mediapurchasingmanagement.SQE@1c4be76]" name="test_2_addSQE_SQL" duration-ms="19" started-at="2015-05-12T15:43:53Z" data-provider="feeder" finished-at="2015-05-12T15:43:53Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[SeleniumTest]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_2_addSQE_SQL -->
        <test-method status="FAIL" signature="test_3_addSQE_errorInfo(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.emar.test.testcases.yiqifabackend.mediapurchasingmanagement.SQE@1c4be76]" name="test_3_addSQE_errorInfo" duration-ms="1" started-at="2015-05-12T15:43:53Z" data-provider="addSQE_fail" finished-at="2015-05-12T15:43:53Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
The data provider is trying to pass 1 parameters but the method com.emar.test.testcases.yiqifabackend.mediapurchasingmanagement.SQE#test_3_addSQE_errorInfo takes 3 and TestNG is unable in inject a suitable object]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
The data provider is trying to pass 1 parameters but the method com.emar.test.testcases.yiqifabackend.mediapurchasingmanagement.SQE#test_3_addSQE_errorInfo takes 3 and TestNG is unable in inject a suitable object
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1361)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1225)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:330)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: java.lang.ArrayIndexOutOfBoundsException: 1
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1359)
	... 17 more
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_3_addSQE_errorInfo -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:com.emar.test.testcases.yiqifabackend.mediapurchasingmanagement.SQE@1c4be76]" name="afterTest" is-config="true" duration-ms="1143" started-at="2015-05-12T15:43:53Z" finished-at="2015-05-12T15:43:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- com.emar.test.testcases.yiqifabackend.mediapurchasingmanagement.SQE -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
